### ISSUE TITLE
Any Plans for Updates?

### ISSUE THREAD

**Issue by ghost** [mjurczyk/grunt-galen#21]
Hi, 
I was looking for this code only as want to automate it but saw a note, 

Note: This module is considered deprecated for Galenframework v2.0.3+ and requires further development. Stale for now.

Do you have any plans for upgrade? or i can still use this with latest galen version? have not tried but just curious to know.

Regards,
Shailesh Dhal

**Comment by dennisfrank** [comment]
Yeah, I wonder the same thing. Does grunt-galen work with Galen 2.3 out of the box?
